### PR TITLE
Refactor project page to show updated information

### PR DESCRIPTION
**Pull request by Tanish-Ranjan**

### CODE REVIEW

File: docs/events/gsoc/2024/gradle-build-server-android.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # GSoC 2024. Gradle Build Server – support for Android projects
 
-This project aims to enhance [Build Server for Gradle](https://github.com/microsoft/build-server-for-gradle) from Microsoft by integrating powerful Android Studio features. Bridging the gap between Android Studio and [Build Server for Gradle](https://github.com/microsoft/build-server-for-gradle), will significantly improve the development experience for many Android developers using IDEs based on [Build Server Protocol (BSP)](https://build-server-protocol.github.io).
+This project aims to enhance [Gradle Build Server](https://github.com/microsoft/build-server-for-gradle) project from Microsoft by integrating powerful Android Studio features. Bridging the build process gap between [Android Studio](https://developer.android.com/studio) and [Gradle Build Server](https://github.com/microsoft/build-server-for-gradle), will significantly improve the development experience for many Android developers using Text Editors/IDEs which utilize the [Build Server Protocol (BSP)](https://build-server-protocol.github.io).
 
 ## Status
-Accepted
+Passed mid-term evaluation
 
 ## Goal
```
Comment (oleg-nenashev) [inline docs/events/gsoc/2024/gradle-build-server-android.md:8]: Some status summary here would be nice. E.g. DONE / In_PROGRESS / PLANNED / CANCELLED . It is fine without it too, you cave it below

File: docs/events/gsoc/2024/gradle-build-server-android.md [hunk 2]
```diff
@@ -31,38 +31,76 @@ As the popularity of Android development grows, the need for efficient build aut
...
-- Integration tests should simulate real-world build scenarios and interactions between different server components.
+- Unit tests thoroughly cover specific components like composite builds logic, language extension downcasting, Gradle Java Home handling logic, and build target generation logic for android projects.
+- Implement integration tests to verify seamless functionality within the Gradle Build Server environment.
+- Integration tests simulate real-world build scenarios and interactions between different server components.
+
+## Demos
+
+### Mid-Term progress
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/UN0AFCLASZA?si=8qeakyCY-zEHnAbL" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (oleg-nenashev) [inline docs/events/gsoc/2024/gradle-build-server-android.md:94]: This Iframe is IMHO too narrow for a video.
The timestamp is also missing, right?

File: docs/events/gsoc/2024/gradle-build-server-android.md [hunk 3]
```diff
@@ -31,38 +31,76 @@ As the popularity of Android development grows, the need for efficient build aut
...
+
+My work on this project, significantly expanded my skill set, encompassing JSON-RPC, Language Server Protocol, Build Server Protocol, Gradle API, Gradle Tooling API, testing methodologies and merging techniques with git. I successfully applied these learnings to produce the mentioned outcomes.
 
 ## Links
 
-- [Project Proposal](https://kotlinlang.org/docs/gsoc-2024.html#gradle-build-server-support-for-android-projects-medium-or-hard-175-hrs-or-350-hrs)
-- [Contributor Proposal](https://gradle-community.slack.com/files/U06R0NWDM4J/F06S9HX9VMZ/gsoc_proposal.pdf)
+- [GSoC Project](https://summerofcode.withgoogle.com/programs/2024/projects/Guphkt1v)
+- [Contributor Proposal](https://summerofcode.withgoogle.com/media/user/26d3ca3d8061/proposal/gAAAAABmsQWz_dGzhC0daKp4GFsORe9U10ZgqSOGgQGSaw30QKtvHfJLzNJaCwX1V636VneDqtGGEHM-hvYcOU0V5sOwwvvE6p7wXyrj7fX7bSUkIH-DRKI=.pdf)
+- [Mid-Term Project Demo](https://youtu.be/UN0AFCLASZA)
```
Comment (oleg-nenashev) [inline docs/events/gsoc/2024/gradle-build-server-android.md:106]: Add a timestamp here

File: docs/events/gsoc/2024/gradle-build-server-android.md [hunk 4]
```diff
@@ -31,38 +31,76 @@ As the popularity of Android development grows, the need for efficient build aut
...
+
+Throughout the program I have collaborated with all of my mentors and gained invaluable insights into open source development - best practices, project management and contributions. They also helped me learn how to explore large project repositories for research.
+
+My work on this project, significantly expanded my skill set, encompassing JSON-RPC, Language Server Protocol, Build Server Protocol, Gradle API, Gradle Tooling API, testing methodologies and merging techniques with git. I successfully applied these learnings to produce the mentioned outcomes.
 
 ## Links
 
-- [Project Proposal](https://kotlinlang.org/docs/gsoc-2024.html#gradle-build-server-support-for-android-projects-medium-or-hard-175-hrs-or-350-hrs)
-- [Contributor Proposal](https://gradle-community.slack.com/files/U06R0NWDM4J/F06S9HX9VMZ/gsoc_proposal.pdf)
+- [GSoC Project](https://summerofcode.withgoogle.com/programs/2024/projects/Guphkt1v)
```
Comment (oleg-nenashev) [inline docs/events/gsoc/2024/gradle-build-server-android.md:104]:
```suggestion
- [Project page on the GSoC site](https://summerofcode.withgoogle.com/programs/2024/projects/Guphkt1v)
```

File: docs/events/gsoc/2024/gradle-build-server-android.md (hunk 1 above)
Comment (Tanish-Ranjan) [reply docs/events/gsoc/2024/gradle-build-server-android.md:8]: Fixed in https://github.com/gradle/community/pull/53/commits/2dff07fafdaa6178b1959e3f2e28aff1572fe6df

File: docs/events/gsoc/2024/gradle-build-server-android.md (hunk 3 above)
Comment (Tanish-Ranjan) [reply docs/events/gsoc/2024/gradle-build-server-android.md]: Fixed in https://github.com/gradle/community/pull/53/commits/2dff07fafdaa6178b1959e3f2e28aff1572fe6df

File: docs/events/gsoc/2024/gradle-build-server-android.md (hunk 2 above)
Comment (Tanish-Ranjan) [reply docs/events/gsoc/2024/gradle-build-server-android.md]: Fixed in https://github.com/gradle/community/pull/53/commits/2dff07fafdaa6178b1959e3f2e28aff1572fe6df

File: docs/events/gsoc/2024/gradle-build-server-android.md (hunk 4 above)
Comment (Tanish-Ranjan) [reply docs/events/gsoc/2024/gradle-build-server-android.md]: Fixed in https://github.com/gradle/community/pull/53/commits/2dff07fafdaa6178b1959e3f2e28aff1572fe6df

**Review by oleg-nenashev — APPROVED** [review]
Thanks a lot!

**Pull request merged**